webapp/src/components/FallbackAvatar/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Avatar, AvatarProps } from '@mui/material';
+
+interface FallbackAvatarProps extends AvatarProps {
+  fallback: string;
+}
+
+export default function FallbackAvatar(props: FallbackAvatarProps) {
+  const { fallback, ...avatarProps } = props;
+
+  function stringToColor(string: string) {
+    let hash = 0;
+    let i;
+
+    for (i = 0; i < string.length; i += 1) {
+      hash = string.charCodeAt(i) + ((hash << 5) - hash);
+    }
+
+    let color = '#';
+
+    for (i = 0; i < 3; i += 1) {
+      const value = (hash >> (i * 8)) & 0xff;
+      color += `00${value.toString(16)}`.slice(-2);
+    }
+
+    return color;
+  }
```

Comment:
maybe extract this util function to a dedicated file? (and add unit test with jest?)